streams/id.m3u

```diff
@@ -23,42 +25,50 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
 http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="IndonesiaChannel.id" status="online",Indonesia Channel (720p) [Not 24/7]
 http://wowzaprod236-i.akamaihd.net/hls/live/1019903/7dd4cf51/playlist.m3u8
-#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
-#EXTINF:-1 tvg-id="" status="online",iNews (720p)
-https://vcdn2.rctiplus.id/live/eds/inews_fta/live_fta/inews_fta.m3u8
-#EXTINF:-1 tvg-id="KompasTV.id" status="error",Kompas TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/874-kompas-tv
+#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar (720p) [Geo-blocked]
+http://rr2.dens.tv/s/s04/01.m3u8
+#EXTINF:-1 tvg-id="INews.id" status="online",iNews (720p)
+https://goldenlivektv.com/Live_TV/Channel_Inews.m3u8
```

Comment:
Redirects to https://inews-linier.rctiplus.id/live/eds/INEWS_2021/live_fta/index.m3u8